index.html

```diff
@@ -2998,17 +2998,16 @@ <h3>Trust Model</h3>
 
         <ul>
           <li>
-The [=verifier=] trusts the [=issuer=] to issue the [=credential=] that
-it received after verification has been performed. 
-To establish this trust, a [=credential=] is expected to either:
+The [=verifier=] trusts the [=issuer=] to have issued the [=credential=] that
+it has received, providing that the [=credential=] has either:
             <ul>
               <li>
-Be secured with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
-[=issuer=] generated the [=credential=] (that is, it is a
+Been secured with a <a href="#securing-mechanisms">securing mechanism</a> establishing
+that the [=issuer=] generated the [=credential=] (that is, it is a
 [=verifiable credential=]), or
               </li>
               <li>
-Have been transmitted in a way clearly establishing that the [=issuer=]
+Been transmitted in a way clearly establishing that the [=issuer=]
```

Comment:
Further editing of this text is probably not worthwhile here, as it gets substantially rephrased in #1474.